src/Module.Server/Common/CrpgUserManagerClient.cs

```diff
@@ -60,7 +61,7 @@ private void HandleUpdateCrpgUser(UpdateCrpgUser message)
         crpgPeer.User = message.User;
         if (crpgPeer.User.ClanMembership != null)
         {
-            crpgPeer.Clan = new CrpgClan { Id = crpgPeer.User.ClanMembership.ClanId };
+            crpgPeer.Clan = new CrpgClan { Id = crpgPeer.User.ClanMembership.ClanId, Name = message.ClanName };
```

Comment:
Would the scoreboard look nicer if we were to just have the clan tag rather than the full name?

Reply:
I thought about this , but for the moment i want to see it with the name. 
I should probably implement a if string too long use tag instead.